Frameworks/UIKit/UICTFont.mm

```diff
@@ -24,6 +24,19 @@ @implementation UIFontPrototype
 
 PROTOTYPE_CLASS_REQUIRED_IMPLS(UICTFont)
 
+// WinObjC-only extension for UINibUnarchiver
+- (instancetype)initWithCoder:(NSCoder*)coder {
+    NSString* name = [coder decodeObjectForKey:@"UIFontName"];
+    if ([name length] < 1) {
```

Comment:
yeah, now that you mention it (this was copied forward from some really old code).  I'll change it to == 0.

---
In reply to: [85619699](https://github.com/Microsoft/WinObjC/pull/1263#discussion_r85619699) [](ancestors = 85619699)